cms/djangoapps/contentstore/utils.py

```diff
@@ -160,7 +160,10 @@ def get_lms_link_for_certificate_web_view(user_id, course_key, mode):
 
 def course_image_url(course):
     """Returns the image url for the course."""
-    loc = StaticContent.compute_location(course.location.course_key, course.course_image)
+    try:
+        loc = StaticContent.compute_location(course.location.course_key, course.course_image)
+    except InvalidKeyError:
+        return None
```

Comment:
What's the desired behavior (on the frontend) when a course is missing an image? Should the image tag be left out? Is there a default image that we should plug in? I think if you answer that question (or product does), then we'll know what the right thing to do in this code is. Also, it would be good to update the docstring for this method to be specific about what it's trying to do.

Finally, looking around, I see that this is the `cms` version of this method, but that there's also a version used in `lms` templates. Do we need to fix that code as well?